include/glob.hpp

```diff
@@ -28,47 +29,82 @@ namespace glob {
 
 		/* Add */
 		vect operator+(const vect &v) const { return vect(x + v.x,
-		                                                  y + v.y); };
+														  y + v.y); };
 
 		/* Subtract */
 		vect operator-(const vect &v) const { return vect(x - v.x,
-		                                                  y - v.y); };
+														  y - v.y); };
 
 		/* Multiply by a scalar */
 		vect operator*(const float &s) const { return vect(x * s,
-		                                                   y * s); };
+														   y * s); };
 
 		/* Divide by a scalar */
 		vect operator/(const float &s) const { return vect(x / s,
-		                                                   y / s); };
+														   y / s); };
 
 		/* Dot product */
-		float operator*(const vect &v) const { return x * v.x
-		                                            + y * v.y; };
+		float operator*(const vect &v) const { return x * v.x + y * v.y; };
 
 		/* Equals? */
 		bool operator==(const vect &v) const { return x == v.x && y == v.y; };
 
 		/* Subtract from this vector */
-		void operator+=(const vect &v) { x += v.x; y += v.y; };
+		void operator+=(const vect &v)
+		{
+			x += v.x;
+			y += v.y;
+		};
 
 		/* Subtract from this vector */
-		void operator-=(const vect &v) { x -= v.x; y -= v.y; };
+		void operator-=(const vect &v)
+		{
+			x -= v.x;
+			y -= v.y;
+		};
 
 		/* Multiply into this vector by a scalar */
-		void operator*=(const float &s) { x *= s; y *= s; };
+		void operator*=(const float &s)
+		{
+			x *= s;
+			y *= s;
+		};
+
+		/* Entry-wise multiply into this vector by another vector */
+		void operator*=(const vect &v)
+		{
+			x *= v.x;
+			y *= v.y;
+		}
 
 		/* Divide into this vector by a scalar */
-		void operator/=(const float &s) { x /= s; y /= s; };
+		void operator/=(const float &s)
+		{
+			x /= s;
+			y /= s;
+		};
```

Comment:
You can break all of these things apart, but don't indent the brackets wrong while doing it.